MIDI Editor/ugurcanorcun_ReaKS.lua

```diff
@@ -437,7 +453,7 @@ local function loop()
                         if ActivatedKS[i] ~= nil then ImGui.PushStyleColor(ctx, ImGui.Col_Button, EnumThemeColors.G) end
 
                         if ImGui.Button(ctx, articulation, 100) then
-                            local isShiftHeld = ImGui.IsKeyDown(ctx, ImGui.Key_LeftShift) or ImGui.IsKeyDown(ctx, ImGui.Key_RightShift)
+                            local isShiftHeld = ImGui.GetKeyMods(ctx) == 8192
```

Comment:
Numeric values are not stable nor part of the API contract for backward compatibility. `ImGui.Mod_Shift` is 8192 today, but it could be something else tomorrow. It was something else a few versions ago.

Only the names (ImGui.Something) should be used.

If you don't care if other modifiers (ctrl/alt/super) are pressed at the same time:
```suggestion
                            local isShiftHeld = ImGui.IsKeyDown(ctx, ImGui.Mod_Shift)
```

If it needs to be exactly Shift and no other modifier:
```suggestion
                            local isShiftHeld = ImGui.GetKeyMods(ctx) == ImGui.Mod_Shift
```

Reply:
So committing suggestions is a feature of GitHub, huh? Terrific! :D
Thanks for the correction :)